package.json

```diff
@@ -91,6 +91,7 @@
     "@storybook/addon-a11y": "6.5.13-alpha.0",
     "@storybook/addon-essentials": "6.5.13-alpha.0",
     "@storybook/react": "6.5.13-alpha.0 ",
+    "@storybook/testing-library": "^0.0.13",
```

Comment:
I tought you said you were going to remove this

Reply:
Oopsie, removed now.